watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -216,6 +217,29 @@ def test_option_pressure_change_calculated():
     assert isinstance(m.fs.unit.eq_area, Constraint)
 
 
+@pytest.mark.unit
+def test_option_friction_factor_spiral_wound():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties = props.NaClParameterBlock()
+    m.fs.unit = OsmoticallyAssistedReverseOsmosis0D(
+        property_package=m.fs.properties,
+        has_pressure_change=True,
+        concentration_polarization_type=ConcentrationPolarizationType.calculated,
+        mass_transfer_coefficient=MassTransferCoefficient.calculated,
+        pressure_change_type=PressureChangeType.calculated,
+        friction_factor=FrictionFactor.spiral_wound,
+    )
+
+    assert m.fs.unit.config.friction_factor == FrictionFactor.spiral_wound
+    assert isinstance(m.fs.unit.feed_side.velocity, Var)
+    assert isinstance(m.fs.unit.feed_side.eq_friction_factor_spiral_wound, Constraint)
+    assert isinstance(m.fs.unit.permeate_side.velocity, Var)
+    assert isinstance(
+        m.fs.unit.permeate_side.eq_friction_factor_spiral_wound, Constraint
```

Comment:
@adam-a-a @MarcusHolly the new OARO test function does check the permeate side ...

Reply:
Yep, I was just saying that the code Adam pointed to in his comment was for the 0D RO file (and not OARO like his comment suggested). Anyways, everything should be good to go now